src/Data/Vec/Properties.agda

```diff
@@ -76,13 +77,84 @@ unfold-take : ∀ n {m} x (xs : Vec A (n + m)) → take (suc n) (x ∷ xs) ≡ x
 unfold-take n x xs with splitAt n xs
 unfold-take n x .(xs ++ ys) | xs , ys , refl = refl
 
+take-distr-zipWith : ∀ {m n} → (f : A → B → C) →
+                     (u : Vec A (m + n)) → (v : Vec B (m + n)) →
```

Comment:
Apologies I didn't pick up on this in the first round of the review, but according to the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md) and in order to keep the these proofs consistent with the rest of this file, vectors should be given the name `xs` and `ys` rather than `u` and `v`.

Reply:
Fixed.